flags.go

```diff
@@ -464,6 +464,9 @@ func populateFlags(c *cli.Context) (flags *flagStorage, err error) {
 
 	clientProtocolString := strings.ToLower(c.String("client-protocol"))
 	clientProtocol := mountpkg.ClientProtocol(clientProtocolString)
+	userStatCacheTTL := util.RoundDurationToNextMultiple(c.Duration("stat-cache-ttl"), time.Second)
```

Comment:
Can we please update the flag description, semantics doc and other places in the repo wherever any of the stat and type cache ttl flags are used or described ?

Asking because it becomes easier to rollback in such cases. (I would also suggest to create internal tickets to update public docs in future as we make these changes)

Reply:
Done. PTAL.